data/complex/basic.lean

```diff
@@ -172,7 +172,7 @@ by refine { zero := 0, add := (+), neg := has_neg.neg, one := 1, mul := (*), ..}
 
 @[simp] lemma bit0_re (z : ℂ) : (bit0 z).re = bit0 z.re := rfl
 @[simp] lemma bit1_re (z : ℂ) : (bit1 z).re = bit1 z.re := rfl
-@[simp] lemma bit0_im (z : ℂ) : (bit0 z).im = bit0 z.im := rfl
+@[simp] lemma bit0_im (z : ℂ) : (bit0 z).im = bit0 z.im := eq.refl _
```

Comment:
It's intended to get rid of the `_refl_lemma` attribute, because `bit1_im` is not `rfl`, and it's a bad `dsimp` lemma if used without `bit1_im`. i.e., `dsimp` would simplify `6.im` to `bit0 3.im`, and then stop.

Reply:
On the other hand, 6 is the smallest number for which this behaviour happens, and I hardly ever use numbers as big as 6.